### PR TITLE
Add modal to confirm logout action

### PR DESCRIPTION
**Pull request by cmarin001**
This pull request introduces a new feature: a modal dialog that prompts users to confirm their logout action. This enhancement aims to improve the user experience by preventing accidental logouts and ensuring users are making deliberate decisions to exit their sessions.

### CODE REVIEW

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/feedback/LogoutModal/LogoutModal.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { StoryFn } from "@storybook/react";
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/LogoutModal.stories.tsx:1]: solved

File: src/components/feedback/LogoutModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { Link } from "react-router-dom";
+
+interface ILogoutModalProps {
+  handleShowBlanket: () => void;
+  logoutPath: string;
+}
+
+function LogoutModal(props: ILogoutModalProps) {
+  const { logoutPath, handleShowBlanket } = props;
+  const isSmallScreen = useMediaQuery("(max-width: 744px)");
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/index.tsx:21]: solved

File: src/components/feedback/LogoutModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+  return (
+    <StyledBackdropBlanket>
+      <Blanket>
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/index.tsx:25]: solved

File: src/components/feedback/LogoutModal/styles.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  theme?: typeof inube;
+  isSmallScreen: boolean;
+}
+
+const StyledBackdropBlanket = styled.div`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  background-color: transparent;
+  z-index: 3;
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/styles.ts:14]: solved

File: src/components/feedback/LogoutModal/styles.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+`;
+
+const StyledModal = styled.div<IStyledModal>`
+  background-color: ${inube.color.surface.light.clear};
+  height: auto;
+  max-height: 180px;
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/styles.ts:20]: solved

File: src/components/feedback/LogoutModal/styles.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+import { inube } from "@inube/design-system";
+
+interface IStyledModal {
+  theme?: typeof inube;
+  isSmallScreen: boolean;
+}
+
+const StyledBackdropBlanket = styled.div`
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/styles.ts:9]: What are the benefits of adopting this approach? Moreover, considering that we're employing non-semantic HTML throughout the application, I believe we should maintain consistency

File: src/components/feedback/LogoutModal/index.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+            </Stack>
+            <Stack justifyContent="flex-end" gap="8px">
+              <Button
+                appearance="gray"
+                spacing={isSmallScreen ? "compact" : "wide"}
+                onClick={handleShowBlanket}
+              >
+                Cancelar
+              </Button>
+              <Link to={logoutPath}>
```
[the remark this answers is not in the document]
Comment (cmarin001) [reply src/components/feedback/LogoutModal/index.tsx:57]: solved

File: src/components/feedback/LogoutModal/LogoutModal.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+import { BrowserRouter } from "react-router-dom";
+import { useState } from "react";
+import { action } from "@storybook/addon-actions";
+import { Button, Stack } from "@inube/design-system";
+import { ILogoutModalProps, LogoutModal } from ".";
+
+const story = {
+  title: "feedback/logout",
```
Comment (carin91) [inline src/components/feedback/LogoutModal/LogoutModal.stories.tsx:9]: The correct way would be "components/feedback/logout"

image

**Review by carin91 — APPROVED** [review]

File: src/components/feedback/LogoutModal/LogoutModal.stories.tsx (hunk 8 above)
Comment (cmarin001) [reply src/components/feedback/LogoutModal/LogoutModal.stories.tsx]: solved

**Pull request merged**